parlai/core/torch_ranker_agent.py

```diff
@@ -65,6 +65,19 @@ def add_cmdline_args(cls, argparser):
             '--ignore-bad-candidates', type='bool', default=False,
             help='Ignore examples for which the label is not present in the '
                  'label candidates. Default behavior results in RuntimeError. ')
+        agent.add_argument(
+            '--embedding-normalization', type='bool', default=False,
+            help='normalization after embeddings for transformer. True or False'
+        )
+        agent.add_argument(
```

Comment:
gelu and normalization should be part of both the ranker and the generator, please!